src/schema/server_settings.py

```diff
@@ -99,87 +99,287 @@ class Difficulty(str, Enum):
+        meta_setting={
+            # Validated
+            "name": "narrative_tone",
+            "label": "Writing Style",
+            "description": "What is the writing style of your story? E.g.: Serious, Silly, Gritty, Film Noir, Heady, etc.",
+            "type": "text",
+            "default": "silly",
+            "suggestOutputType": "narrative_tone",
+        }
     )
 
     adventure_background: Optional[str] = Field(
-        description="Description of the background setting in which the adventure will take place.  Can include descriptions of genre, characters, specific items and locations that exist in the world, references to real-world things, etc."
+        meta_setting={
+            # Validated
+            "name": "adventure_background",
+            "label": "Adventure Background",
+            "description": """Description of the background setting in which the adventure will take place.
+
+Can include descriptions of genre, characters, specific items and locations that exist in the world, references to real-world things, etc.""",
+            "type": "longtext",
+            "default": "A fantasy world",
+            "suggestOutputType": "adventure_background",
+        },
     )
 
     narrative_voice: Optional[str] = Field(
-        description="What is the narrative voice of the adventure?  Some possibilities: children’s book, young adult novel, fanfic, high literature"
+        meta_setting={
+            # Validated
+            "name": "narrative_voice",
+            "label": "Genre",
+            "description": "What is the genre of your story? E.g.: children’s book, young adult novel, fanfic, high literature.",
+            "type": "text",
+            "default": "young adult novel",
+            "suggestOutputType": "narrative_voice",
+        },
     )
 
     adventure_goal: str = Field(
         default="To rid the world of evil",
-        description="What is the ultimate goal / motivation of this story?",
+        meta_setting={
+            # Validated
+            "name": "adventure_goal",
+            "label": "Adventure Goal",
+            "description": "What is the ultimate goal / motivation of this adventure?",
+            "type": "longtext",
+            "default": "To rid the world of evil",
+            "suggestOutputType": "adventure_goal",
+        },
     )
 
     fixed_quest_arc: Optional[List[QuestDescription]] = Field(
         default=None,
-        description="If you wish for your adventure to have a fixed set of quests, define them here.",
+        meta_setting={
+            # VALIDATED
+            "name": "fixed_quest_arc",
+            "label": "Fixed Quest Arc",
+            "description": "Optional. If you wish for your adventure to have a fixed set of quests, define them here.",
+            "type": "list",
+            "listof": "object",
+            "listSchema": [
+                # TODO FUTURE this could be pulled directly from QuestDescription
```

Comment:
Yeah I agree -- I think with the character settings too. Maybe we could do that in a next pass?

Reply:
This is another existing comment IIRC from the web, but yeah we can do this.